### DISCUSSION TITLE
Panner HTRF is not precise

### DISCUSSION THREAD

**Discussion by JSmithOner** [WebAudio/web-audio-api#2615] [Q&A]
I've tried implementing the panner in `HTRF` mode wiht THREE.js.
As this point I don't understand all parameters.
But one thing I've experienced using [the example](https://mdn.github.io/webaudio-examples/spatialization/) was that based on the orientation of the speakers the sound settings (maybe phase  )was  abruptly changing, adding cracks during the change.
If you play a bit with the example (and this may only be due to the example code and not the HTF implementation) it looked like the position and orientation is kinda lost sometimes, not reflecting the sound we should hear.
Any feedbacks on this situation and potential updates or fixes would be greatly appreciated

**Comment by hoch** [comment]
@JSmithOner Given that your comments were mostly about the audio quality, I'd like to move this question to the discussion board. Also please feel free to open a new bug on the implementors' issue tracker.

**Comment by JSmithOner** [comment]
@hoch the ticket wasn't only about audio quality it was also the fact that the sound heared wasn't reflecting the sensation that it should based on the speakers position.Any idea, have you noticed the same thing?
